playbooks/fedora-31/setup-rpm-env.yaml

```diff
@@ -5,20 +5,15 @@
       become: yes
       package:
         name:
-          - golang-github-cpuguy83-md2man
-          - ninja-build
-          - meson
           - flatpak-session-helper
           - systemd
           - bats
-          - bash-completion
           - udisks2
           - podman
+          - golang
 
     - name: Setup environment (create missing /run/media)
       command: sudo systemd-tmpfiles --create
 
     - name: Check versions of crucial packages
-      command: rpm -q podman crun conmon fuse-overlayfs flatpak-session-helper
-
-    - include_tasks: ./pre-common.yaml
+      command: rpm -q podman crun conmon fuse-overlayfs flatpak-session-helper golang
```

Comment:
Might you consider adding CRIU so we can create snapshots/container state that can survive a reboot and/get moved between hosts?

Reply:
This is not related to this PR. For a feature request open a new issue, please :).